### DISCUSSION TITLE
Browser Chrome im Kiosk Modus aktualisieren

### DISCUSSION THREAD

**Discussion by Supermario-21** [Zefau/ioBroker.jarvis#1872] [General]
Hallo!
Leider hängt sich der Browser immer wieder auf? Warum auch immer?!

Gibt es eine Möglichkeit diesen per Button in Jarvis oder per GPIO am Raspi zu aktualisieren?

Ich hatte es so probiert im Jarvis, leider ohne Erfolg.

Danke im Voraus

![image](https://user-images.githubusercontent.com/104907692/189629505-b8bbc2cb-a002-41fb-96ae-d54d8b2e362c.png)

![image](https://user-images.githubusercontent.com/104907692/189629605-310d1cb6-78fd-428f-b0ad-63621092b724.png)

![image](https://user-images.githubusercontent.com/104907692/189636008-0c16c644-0d0e-4a5a-9296-62928a4786d1.png)

**Comment by Zefau** [comment]
> Leider hängt sich der Browser immer wieder auf? Warum auch immer?!

Was meinst du denn mit "aufhängen"? Reagiert der Browser nicht mehr aus Performance-Gründen? Oder aktualisieren die Werte lediglich nicht?

Ist es mit der aktuellen alpha noch immer so?

Wenn du Pro bist kannst du direkt in jarvis Skripte integrieren und dort via `setTimeout` einen Reload einplanen.

**Reply by Supermario-21** [reply]
Hallo! Ich nutze v3.0.13! Ich habe Touch 7" und iPads und iPhones im Einsatz. Bei den Apple Geräten kommt das gar nicht vor (Safari, Edge, kein Chrome!?). Am Touch (Chrom-Kiosk) leider schon öfters.  Die Wertfelder werden im Browser mit einem blauen "Dauerrad" angezeigt. Das ändert sich auch nach Stunden nicht. Wenn ich die Jarvis-Instanz neu starte, bleibt das trotzdem so. Mit F5 funktioniert es wieder. Möchte aber dort keine Tastatur liegen haben. Im Habpanel gab es einen Aktualisierungbutton. Die Pro-Version habe ich noch nicht.

**Reply by Zefau** [reply]
Geh mal auf die Alpha und schau, ob es dort auch in auftritt. Die Alpha ist inzwischen schon sehr belastbar und geht zeitnah in die Beta.
